kyuubi-common/src/main/scala/org/apache/kyuubi/service/FrontendService.scala

```diff
@@ -41,4 +41,9 @@ trait FrontendService {
    * An optional `ServiceDiscovery` for [[FrontendService]] to expose itself
    */
   val discoveryService: Option[Service]
+
+  /**
+   * An optional `UI`for each engine expose it's ui
+   */
+  def ui: Option[String] = None
```

Comment:
How about changing it to 
```
def extraConfs: Map[String, String] = Map.empty
```
?
We may want to expose more information in the future, e.g. metrics

cc @turboFei @yaooqinn @cxzl25 @wForget

Reply:
+1 for 
```
def extraConfs: Map[String, String] = Map.empty
```

Reply:
Thanks for suggestion, it has been revised.